pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/OptionsTest.java

```diff
@@ -74,6 +74,8 @@ public void multipleProperties() throws Exception {
 
         WorkflowJob p = b.getParent();
 
+        // We test for skipDefaultCheckout() in the Jenkinsfile itself by verifying that Jenkinsfile isn't in the workspace
```

Comment:
If you look at `src/test/resources/multipleProperties.groovy`, you can see that it includes a `sh "test -f Jenkinsfile"` call - i.e., if there's a `Jenkinsfile` in the workspace, the build will pass. Which isn't the desired result - it should fail, since we skipped the default checkout, and it does, as expected.